token/delegation/examples_test.go

```diff
@@ -123,40 +146,56 @@ func ExampleNew() {
 // - a delegation.Token with the subject (sub) set to the value of issuer
 // (iss).
 func ExampleRoot() {
-	issuerPrivKey := examplePrivKey(issuerPrivKeyCfg)
-	audienceDID := exampleDID(AudienceDID)
-	command := exampleCommand(subJectCmd)
-	policy := examplePolicy(subjectPol)
-
-	// Don't do this in your code - a nonce should be a cryptographically
-	// strong random slice of bytes to ensure the integrity of your private
-	// key.  For this example, a fixed nonce is required to obtain the fixed
-	// printed output (below).  If unsure of what value to supply for the
-	// nonce, don't pass the WithNonce option and one will be generated
-	// when the token is created.
-	nonce := []byte{0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0a, 0x0b}
-
-	tkn, err := delegation.Root(
-		issuerPrivKey,
-		audienceDID,
-		command,
-		policy,
-		delegation.WithNonce(nonce),
+	issPriv, issPub, err := crypto.GenerateEd25519Key(rand.Reader)
```

Comment:
How do the examples pass when you don't have fixed input?

Reply:
it doesn't check the values, but that's ok. What we need is a "quick start", not a test.